### PR TITLE
fix(RequestUi):BU & ProjectName not loading for Closed CR table

### PR DESCRIPTION
**Pull request by akapti**
[//]: # (Copyright Bosch.IO GmbH 2020)
[//]: # (This program and the accompanying materials are made)
[//]: # (available under the terms of the Eclipse Public License 2.0)
[//]: # (which is available at https://www.eclipse.org/legal/epl-2.0/)
[//]: # (SPDX-License-Identifier: EPL-2.0)

> Please provide a summary of your changes here.

> Which issue is this pull request belonging to and how is it solving it?
- #1292 

> Did you add or update any new dependencies that are required for your change?
- No.

Issue: #1292 

### How To Test?

> How should these changes be tested by the reviewer?

- `Project name`, and `business unit` should load for closed clearing request tables, If there are more CR in closed state.
- `Clearing Team` should be able to update the clearing request even if `Agreed Clearing Date` is passed.

> Have you implemented any additional tests?

- No.

### Checklist
Must:
- [x] All related issues are referenced in commit messages and in PR

Signed-off-by: Abdul Kapti <abdul.kapti@siemens-healthineers.com>

### CODE REVIEW

**Review by JaideepPalit — CHANGES_REQUESTED** [review]
Reviewed the PR.
Apart from minor review comment, code looks fine to me.

File: backend/src/src-projects/src/main/java/org/eclipse/sw360/projects/ProjectHandler.java [hunk 1]
```diff
@@ -279,6 +279,8 @@ public RequestStatus deleteProject(String id, User user) throws TException {
 
     @Override
     public List<Project> fillClearingStateSummary(List<Project> projects, User user) throws TException {
+        assertNotEmpty(projects, "Invalid project Ids!");
```
Comment (JaideepPalit) [inline backend/src/src-projects/src/main/java/org/eclipse/sw360/projects/ProjectHandler.java:282]: Not sure, pls check whether assertion for empty project list is required and does not harm other endpoints calling the api,

image

I guess , empty list is handled in `ProjectDatabaseHandler` - return empty list

**Review by JaideepPalit — CHANGES_REQUESTED** [review]

**Comment by akapti** [comment]
> Reviewed the PR.
> Apart from minor review comment, code looks fine to me.

Thanks for the review. Incorporated the requested changes.

**Comment by akapti** [comment]
also added fix for `Open Clearing Request` table not loading.

**Review by JaideepPalit — APPROVED** [review]
Tested successfully on stage env.

**Pull request merged**